arrow-libs/fx/arrow-fx-coroutines/src/commonMain/kotlin/arrow/fx/coroutines/Resource.kt

```diff
@@ -274,6 +291,48 @@ public sealed class Resource<out A> {
       }
     }
 
+  public fun <B, C> parZip(fb: Resource<B>, f: (A, B) -> C): Resource<C> =
+    parZip(Dispatchers.Default, fb, f)
+
+  /**
```

Comment:
Perhaps we should add similar comments with examples to other members in the data type that are public https://arrow-kt.io/docs/apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-resource/map.html